crates/sui-storage/src/indexes.rs

```diff
@@ -148,6 +166,18 @@ fn dynamic_field_index_table_default_config() -> DBOptions {
 fn index_table_default_config() -> DBOptions {
     default_db_options()
 }
+fn coin_index_table_default_config() -> DBOptions {
+    DBOptions {
+        options: optimized_for_high_throughput_options(
+            read_size_from_env(ENV_VAR_COIN_INDEX_BLOCK_CACHE_SIZE_MB).unwrap_or(5 * 1024),
```

Comment:
Is the default value 5KB enough?

Reply:
this is 5GB. the size is in MB as `ENV_VAR_COIN_INDEX_BLOCK_CACHE_SIZE_MB` says